api/Dockerfile

```
@@ -1,4 +1,6 @@
-FROM node:14
+FROM node:20-alpine
+
+RUN apk add bash
```

Build does not work for me without this:

```suggestion
RUN apk add --no-cache bash python3 make g++ \
    && ln -sf python3 /usr/bin/python
```